modules/cluster-logging-updating-logging-to-5-1.adoc

```diff
@@ -0,0 +1,214 @@
+= Updating from OpenShift Logging version 5.0 to 5.1
+
+To update from OpenShift Logging version 5.0 to 5.1, you change the subscriptions for the OpenShift Elasticsearch Operator and Red Hat OpenShift Logging Operator.
+
+It is important that you update the OpenShift Elasticsearch Operator _before_ you update the Red Hat OpenShift Logging Operator. It is also important that you update _both_ Operators.
+
+Otherwise, if you update the operators in the wrong order, Kibana does not update and the Kibana custom resource (CR) is not created. To work around this problem, you delete the Red Hat OpenShift Logging Operator pod. When the Red Hat OpenShift Logging Operator pod redeploys, the Kibana CR is created. When you have updated both Operators correctly, Kibana becomes available again.
+
+.Prerequisites
+
+* The {product-title} version is 4.7 or later.
+
+* The OpenShift Logging status is healthy:
++
+** All pods are `ready`.
+** The Elasticsearch cluster is healthy.
+
+* Your Elasticsearch and Kibana data is backed up.
+
+.Procedure
+
+. Update the OpenShift Elasticsearch Operator:
+
+.. From the web console, click *Operators* -> *Installed Operators*.
+
+.. Select the `openshift-operators-redhat` project.
+
+.. Click the *OpenShift Elasticsearch Operator*.
+
+.. Click *Subscription* -> *Channel*.
+
+.. In the *Change Subscription Update Channel* window, select *stable-5.1* and click *Save*.
+
+.. Wait for a few seconds, then click *Operators* -> *Installed Operators*.
++
+Verify that the OpenShift Elasticsearch Operator version is 5.1.x.
++
+Wait for the *Status* field to report *Succeeded*.
+
+. Update the Red Hat OpenShift Logging Operator:
+
+.. From the web console, click *Operators* -> *Installed Operators*.
+
+.. Select the `openshift-logging` project.
+
+.. Click the *Red Hat OpenShift Logging Operator*.
+
+.. Click *Subscription* -> *Channel*.
+
+.. In the *Change Subscription Update Channel* window, select *stable-5.1* and click *Save*.
+
+.. Wait for a few seconds, then click *Operators* -> *Installed Operators*.
++
+Verify that the Red Hat OpenShift Logging Operator version is 5.1.x.
++
+Wait for the *Status* field to report *Succeeded*.
+
+. Check the logging components:
+
+.. Ensure that all Elasticsearch pods are in the *Ready* status:
++
+[source,terminal]
+----
+$ oc get pod -n openshift-logging --selector component=elasticsearch
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                            READY   STATUS    RESTARTS   AGE
+elasticsearch-cdm-1pbrl44l-1-55b7546f4c-mshhk   2/2     Running   0          31m
+elasticsearch-cdm-1pbrl44l-2-5c6d87589f-gx5hk   2/2     Running   0          30m
+elasticsearch-cdm-1pbrl44l-3-88df5d47-m45jc     2/2     Running   0          29m
+----
++
+.. Ensure that the Elasticsearch cluster is healthy:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch elasticsearch-cdm-1pbrl44l-1-55b7546f4c-mshhk -- es_cluster_health
```

Comment:
"health" instead of "es_cluster_health" returns the column form of this command

Reply:
Updated. Thanks.